thegame/engine/engine.py

```diff
@@ -9,18 +10,32 @@
 
 class Engine:
     def __init__(self, game: BaseGame):
-        self.width = 240
-        self.height = 180
-        self.size = self.width, self.height
+
         self.running = False
-        self.mouse_down_pos = None
+
+        self.display = None
         self.context = game
+        self.width = game.screen_width
+        self.height = game.screen_height
+        self.size = self.width, self.height
+        self.mouse_down_pos = None
 
         pygame.init()
-        pygame.display.set_caption("TheGame")
-        pygame.display.set_mode(self.size)
 
     def start(self):
+
+        pygame.display.set_caption("TheGame")
```

Comment:
This is in the engine, it should probably get the name from ```context``` (the actual game object) so that it's settable.

Reply:
Will do

Reply:
Addressed with https://github.com/Ross-Alexandra/TheGame/pull/16/commits/1711bd575b771a88304b64123375c7bdb693ac66